tests/5.0/target_teams_distribute/test_target_teams_distribute_reduction_and.c

```diff
@@ -0,0 +1,88 @@
+//===--- test_target_teams_distribute_reduction_and.c------------------------===//
+//
+// OpenMP API Version 4.5 Nov 2015
+//
+// This test uses the reduction clause on a target teams distribute directive,
+// testing that the variable in the reduction clause is properly reduced using
+// the and operator.
+//
+////===----------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+#include <math.h>
+
+#define N 1024
+
+int test_and() {
+  char a[N];
+  char result;
+  char host_result;
+      // The below calculation is meant to ensure about half the test arrays will
```

Comment:
The description is slightly confusing. It's supposed to be the case that roughly half of all the different arrays we will test will have all true elements, while the other half will have a few falses. This will mean half of the test iterations (loop on line 33) will give a true result, and the other half false.

Reply:
Edited the description.